acceptance-tests/features/check-elements-width.feature

```diff
@@ -1,5 +1,5 @@
 Feature: Check elements width
 
   Scenario: Check that the input and button have the same width
-    When I go to "http://localhost:8080"
```

Comment:
Please put the base url in a variable so it can be reused across all the places you changed it